src/app/components/organisms/location.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import SectionTitle from '../molecules/SectionTitle/section-title';
+import MapAtom from '../atoms/Map/map';
+import RoundedButton from '../atoms/Button/rounded-button';
+
+const places = [
+  {
+    lat: 20.913533166035844,
+    lng: -100.74378055060164,
+    name: 'Parroquia Centro'
+  },
+  {
+    lat: 20.898342336684696,
+    lng: -100.72659947263716,
+    name: 'Plaza la Luciérnaga'
+  },
+  {
+    lat: 20.93124265594304, 
+    lng: -100.6881409531946,
+    name: 'Splash Inn San Miguel'
+  },
+]
+
+const center = {
+    lat: 20.927441183582008,
+    lng: -100.7104233679535,
+    name: 'Glamping San Miguel'
+  };
+
+
+
+export default function Location() {
+  
+  return (
+    <section id="location" className='flex flex-col items-center mx-5  md:mx-10 max-w-md sm:max-w-xl md:max-w-2xl lg:max-w-5xl xl:max-w-7xl'>
+      <div className="flex flex-col gap-4 sm:gap-6 md:gap-10 lg:gap-16 xl:gap-20 lg:flex-row">
+
+        <div className='flex flex-col gap-10 justify-center w-full lg:w-1/3'>

```

Comment:
Aquí siento que el gap es demasiadooo jajajaja y habría que bajarlo un poco

```suggestion
        <div className='flex flex-col gap-4 sm:gap-5 md:gap-6 justify-center w-full lg:w-1/3'>
```